app/views/users/dashboard.html.erb

```diff
@@ -0,0 +1,95 @@
+<div class="flex flex-col bg-slate-900 items-center justify-center">
+<div class="flex flex-row border-b border-solid border-white w-full">
+
+<div class="flex flex-col bg-slate-500 ml-2 mr-2 text-center rounded items-center w-1/2 m-6 p-3 pl-3 pr-3 overflow-hidden whitespace-no-wrap overflow-ellipsis">
+
+<%= render "partials/dashboard/profile_picture", image_path: "test.jpg", alt_text: "Placeholder" %>
+
+<h1> <%= @user %> </h1>
+
+<%= render "partials/dashboard/text_fields", value: "Lorem Ipsum of a big description of someone that has a lot to say about themselves bla bla bla bla bla bla yada yada" %>
+
+
+</div>
+<div class="flex flex-col bg-slate-300 ml-2 mr-2 rounded w-1/2 m-6 justify-center">
+
+<div class="flex flex-col bg-slate-300 ml-2 mr-2 rounded m-6 text-center">
+
+<%= render "partials/dashboard/h3", label: "X", value: @user_X %>
+<%= render "partials/dashboard/h3", label: "Instagram", value: @user_instagram %>
```

Comment:
Add the migration on user to get the required fields then make this work properly using the correct fields on our current user.

Also, make these fields only show up if the user has indeed added an X / Instagram / Pinterest to their account